pkgs/development/python-modules/authheaders/default.nix

```diff
@@ -0,0 +1,19 @@
+{ buildPythonPackage, fetchPypi, lib, authres, dnspython, dkimpy, publicsuffix }:
+
+buildPythonPackage rec {
+  pname = "authheaders";
+  version = "0.12.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hf1p6ws3jma608pmcb5qsl58xg33wz2s51qqzi9zix0llcnyc97";
+  };
+
+  propagatedBuildInputs = [ authres dnspython dkimpy publicsuffix ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ 
    authres
    dnspython
    dkimpy
    publicsuffix
    ] ++ lib.optionals isPy27 [ ipaddress ];
```
to avoid:
```
  Requirement already satisfied: authres>=1.2.0 in /nix/store/7bxi2wk1p6ppzc8ir9m63ga5xdm2kazn-python2.7-authres-1.2.0/lib/python2.7/site-packages (from authheaders==0.12.1) (1.2.0)
  ERROR: Could not find a version that satisfies the requirement ipaddress (from authheaders==0.12.1) (from versions: none)
  ERROR: No matching distribution found for ipaddress (from authheaders==0.12.1)
```